src/snowflake/cli/plugins/stage/manager.py

```diff
@@ -83,6 +84,16 @@ def get_full_stage_path(self, path: str):
             return prefix + "." + path
         return path
 
+    def get_standard_stage_path(self) -> str:
+        path = self.path
+        return f"@{path}{'/'if self.is_directory and not path.endswith('/') else ''}"
+
+    def get_standard_stage_directory_path(self) -> str:
+        path = self.get_standard_stage_path()
+        if not path.endswith("/"):
+            return path + "/"
+        return path
```

Comment:
I don't know if it's overcomplicated, once we need what user passed, other time we need with / at the end.